tests.sh

```diff
@@ -248,16 +248,6 @@ build_java_linkage_monitor() {
   # Linkage Monitor uses $HOME/.m2 local repository
   MVN="mvn -e -B -Dhttps.protocols=TLSv1.2"
   cd java
-  # Sets java artifact version with SNAPSHOT, as Linkage Monitor looks for SNAPSHOT versions.
-  # Example: "3.9.0" (without 'rc')
-  VERSION=`grep '<version>' pom.xml |head -1 |perl -nle 'print $1 if m/<version>(\d+\.\d+.\d+)/'`
-  cd bom
-  # This local installation avoids the problem caused by a new version not yet in Maven Central
```

Comment:
This local installation is no longer needed. The problem (#6627 ) happened for "version:set" operation, which this PR removes.